src/java.base/share/classes/javax/crypto/CipherSpi.java

```diff
@@ -442,7 +443,7 @@ protected abstract void engineInit(int opmode, Key key,
      *
      * <p>If this cipher (including its underlying feedback or padding scheme)
      * requires any random bytes (e.g., for parameter generation), it will get
-     * them from <code>random</code>.
+     * them from {@code random}.
      *
      * <p>Note that when a Cipher object is initialized, it loses all
      * previously-acquired state. In other words, initializing a Cipher is
```

Comment:
Two `{@code Cipher}` above.

Reply:
Yes, I will look through the whole class (and CipherSpi.java too) regarding the Cipher vs cipher thing.